cloudferrylib/os/actions/check_cloud.py

```diff
@@ -17,8 +17,10 @@
 from keystoneclient.openstack.common.apiclient import exceptions as ks_exc
 from novaclient import exceptions as nova_exc
 from cloudferrylib.base import exception
+import time
```

Comment:
Please put it in the alphabetical order according to this document:
http://docs.openstack.org/developer/hacking/#import-order-template